docs/topics/strings.md

```diff
@@ -158,27 +158,37 @@ To format a string to your specific requirements, use the [`String.format()`](ht
 function. 
 
 The `String.format()` function accepts a format string and one or more arguments. The format string contains one placeholder 
-(`%`) for each remaining argument, followed by [format specifiers](https://docs.oracle.com/javase/8/docs/api/java/util/Formatter.html#summary). 
-Format specifiers are formatting instructions for the respective argument. In the output, each argument fills its 
-corresponding placeholder in the defined format:
+(`%`) for each remaining argument, followed by format specifiers. 
+Format specifiers are formatting instructions for the respective argument, consisting of flags, width, precision, and 
```

Comment:
We can go for shorter paragraphs placing this explanation in a new one. The meaning doesn't change, and we help users scan info.

Reply:
I agree it might help with scanning, but I think it would be more beneficial to keep it as one paragraph (it's still concise) and the information flows logically from introducing the String.format() function to detailing its components (placeholders and format specifiers).